src/Console/Command/CompareCommand.php

```diff
@@ -0,0 +1,323 @@
+            sprintf("Risky (%s)\nAre hidden: %s", count($this->riskyFixers), $this->hideRisky ? self::YES : self::NO),
+            sprintf("Inherited (%s)\nAre hidden: %s", $this->countInherited, $this->hideInherited ? self::YES : self::NO),
+            'In RuleSet',
+        ];
+
+        $table->setHeaders([$columns]);
+
+        $table->setRows($this->filterRows());
+
+        return $table;
+    }
+
+    /**
+     * @return array
+     */
+    private function filterRows()
+    {
+        $hideConfigured = $this->hideConfigured;
+        $hideEnabled = $this->hideEnabled;
+        $hideRisky = $this->hideRisky;
+        $hideInherited = $this->hideInherited;
+
+        $rows = array_filter($this->list, function (array $fixer) use ($hideConfigured, $hideEnabled, $hideRisky, $hideInherited) {
+            if ($hideConfigured && $fixer['is_configured']) {
+                return false;
+            }
+
+            if ($hideEnabled && $fixer['is_enabled']) {
+                return false;
+            }
+
+            if ($hideRisky && $fixer['is_risky']) {
+                return false;
+            }
+
+            if ($hideInherited && $fixer['is_inherited']) {
+                return false;
+            }
+
+            return true;
+        });
+
+        $this->dump = $rows;
+
+        return array_map(function (array $fixer) {
+            $path = '.php_cs';
+            if (isset($fixer['in_set'])) {
+                $path = implode(' > ', array_reverse($fixer['in_set']));
+            }
+
+            return [
+                'name' => $fixer['name'],
+                'is_configured' => $fixer['is_configured'] ? self::YES : self::NO,
+                'is_enabled' => $fixer['is_enabled'] ? self::YES : self::NO,
+                'is_risky' => $fixer['is_risky'] ? self::YES : self::NO,
+                'is_inherited' => $fixer['is_inherited'] ? self::YES : self::NO,
+                'in_set' => $path,
+            ];
+        }, $rows);
+    }
+
+    /**
+     * @param OutputInterface $output
+     */
+    private function dump(OutputInterface $output)
+    {
+        $line = empty($this->dump)
+            ? 'You are aware of all exsisting rules! Yeah!'
+            : var_export(
+                array_map(function () {return false; }, $this->dump),
+                true
+            );
+
+        $output->writeln(
+            empty($this->dump)
+                ? $line
+                : "\nCopy and paste the following rules in your .php_cs file:\n\n"
+                .'\\\\ Below the rules I don\'t want to use'."\n"
+                .'\''.$line."\n"
+                .'\\\\ END Rules to never use'."\n"
+        );
+    }
+
+    /**
+     * @param FixerInterface $fixer
+     *
+     * @return bool
+     */
+    private function isFixerConfigured(FixerInterface $fixer)
+    {
+        return array_key_exists($fixer->getName(), $this->configuredFixers);
```

Comment:
depends on scenario, `array_key_exists` is 4-10k (yes, k!) times slower. don't use it if you don't need to

Reply:
Just because I'm curious: have you any test to prove this? Anyway I trust you, so I'm going to change this.

Reply:
DIY: https://www.google.pl/search?q=isset+vs+array_key_exists+performance&oq=isset+vs+array_key_exists+performance&aqs=chrome..69i57j69i60l3.6926j0j7&sourceid=chrome&ie=UTF-8

a lot of analysis were executed. you need to get outcome of them with caution - a performance difference depends very much on given test, dataset and user php version.
pick one and execute yourself, eg on 3v4l.
one thing is constant - `isset` is **always** faster